__tests__/app.test.js

```diff
@@ -133,3 +133,25 @@ describe("5. GET /api/articles returns 200", () => {
       });
   });
 });
+
+describe("6. GET /api/articles/:article_id/comments returns 200", () => {
+    describe("/api/articles/:article_id/comments", () => {
+      describe("GET /api/articles/1/comments returns 200", () => {
+        test("status code: 200", () => {
+          return request(app).get("/api/articles/1/comments").expect(200);
+        });
```

Comment:
I'd recommend adding a test to check the expected properties. This can be done by using jest matches such as **toMatchObject or ObjectContaining.**